api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -91,15 +94,24 @@ public WorkspacesApi workspacesApiAcls(
     return api;
   }
 
-  @Bean
+  @Bean(name = END_USER_STATIC_NOTEBOOKS_API)
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
-  public StaticNotebooksApi staticNotebooksApi(
+  public StaticNotebooksApi endUserStaticNotebooksApi(
       @Qualifier(END_USER_API_CLIENT) ApiClient apiClient) {
     StaticNotebooksApi api = new StaticNotebooksApi();
     api.setApiClient(apiClient);
     return api;
   }
 
+  @Bean(name = SERVICE_ACCOUNT_STATIC_NOTEBOOKS_API)
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public StaticNotebooksApi serviceAccountStaticNotebooksApi(
+      @Qualifier(SERVICE_ACCOUNT_API_CLIENT) ApiClient apiClient) {
+    StaticNotebooksApi api = new StaticNotebooksApi();
```

Comment:
Looks like these three lines could be their own helper function called by both beans.

Reply:
I'm not sure that gets us much

Reply:
it's easy to do and makes it obvious that it's the same bean but with different ApiClients.